src/helpers/ApiCallHelper.tsx

```diff
@@ -5,3 +5,15 @@ export const fetchStations = () => {
         },
     });
 };
+
+export const fetchFares = (originId: string, destinationId: string) => {
+    const dateTime = '2023-07-13T12:16:27.371Z';
+    const numberOfAdults = 2;
+    const numberOfChildren = 0;
+    return fetch(
```

Comment:
Do you recon you can make a function with the X-API-KEY header so that future API calls are easier?
Also if we have to change the way API calls are authenticated, we want to only have to change it in one place